src/components/FreakTile/__test__/FreakTile.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import FreakTile from '../FreakTile';
+
+describe('FreakTile', () => {
+  it('should render the freak tile', () => {
+    render(<FreakTile id={ 5 } picture="some-url" name="naruto" />);
+
+    expect(screen.getByTestId('freak-tile')).toBeInTheDocument();
+  });
+
+  it('should render an hyperlink to the freak page', () => {
+    const id = 6;
+    render(<FreakTile id={ id } picture="some-url" name="naruto" />);
+
+    expect(screen.getByTestId('freak-tile')).toHaveAttribute('href', `/freaks/${ id }`);
+  });
+});
```

Comment:
We should also test that the name appears correctly, write a test for this as well. Would also like to see a test for the picture, but I don't know how we could check that.